INSTALL

```diff
@@ -11,7 +11,7 @@ Install from Source:
 
 3. Run ./autogen.sh.
 
-4. Run ./configure.
+4. Run ./configure [--enable-tirpc=yes/no] [--disable-silent-rules] ....
```

Comment:
@lxbsz also change the point 1, to reflect new targetcli recommended version.

Reply:
Fixed it. Thanks.